src/views/partials/head.hbs

```diff
@@ -2,5 +2,8 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="/css/signup.css">
```

Comment:
so you want the `signup.css` file to show on each page?

@amd711   did you really review the pull request?